pkg/app/piped/platformprovider/lambda/lambda.go

```diff
@@ -25,6 +25,7 @@ import (
 
```

Comment:
nit: It would be nice to delete a line break between "golang.org/x/sync/singleflight" and "github.com/aws/aws-sdk-go-v2/service/lambda"

Reply:
Thank you!
Fixed this and similars on this commit.
https://github.com/pipe-cd/pipecd/pull/5434/commits/b343897b76f419cf4a55e22432ebd0475da1ba9d